notebook/lab/handlers.py

```diff
@@ -18,12 +19,12 @@ def get(self):
             terminals_available=self.settings['terminals_available'],
             mathjax_url=self.mathjax_url))
 
-
 #-----------------------------------------------------------------------------
 # URL to handler mappings
 #-----------------------------------------------------------------------------
 
-
 default_handlers = [
     (r"/lab", LabHandler),
+    (r"/lab/(.*)", AuthenticatedFileHandler,
```

Comment:
I suspect this wants to be a regular StaticFileHandler (or more precisely FileFindHandler, to get identical caching behavior to the existing static files). We don't normally authenticate access to the basic js/css.